tests/rptest/clients/rpk.py

```diff
@@ -666,35 +606,115 @@ def try_describe_group(group):
                     raise
 
             lines = out.splitlines()
+            group_name = None
+            coordinator = -1
+            state = None
+            balancer = None
+            members = None
+            total_lag = None
+            last_field = 0
+            for idx, l in enumerate(lines):
+                field = try_parse_field(l)
+
+                if field is None:
+                    continue
+                name, value = field
+
+                if name == "GROUP":
+                    group_name = value
+                elif name == "COORDINATOR":
+                    coordinator = int(value)
+                elif name == "STATE":
+                    state = value
+                elif name == "BALANCER":
+                    balancer = value
+                elif name == "MEMBERS":
+                    members = int(value)
+                elif name == "TOTAL-LAG":
+                    total_lag = int(value)
+                else:
+                    self._redpanda.logger.error(
```

Comment:
we should assert instead of logging an error (people ignore errors if they not in the way)